plugin-build/build.gradle.kts

```diff
@@ -61,6 +62,11 @@ dependencies {
     compileOnly(Libs.ASM)
     compileOnly(Libs.ASM_COMMONS)
 
+    implementation(kotlin("gradle-plugin-api"))
```

Comment:
I think it's the same as the last one here:
```
implementation("org.jetbrains.kotlin:kotlin-gradle-plugin-api:${KotlinCompilerVersion.VERSION}")
```
so we could remove either of them

Reply:
Also, it seems like we don't need `-api` at all, I haven't found usages of it in `plugin-build`